### DISCUSSION TITLE
MMS with ts-jest typegoose setup

### DISCUSSION THREAD

**Discussion by Sed9yDataBank** [typegoose/mongodb-memory-server#519] [Q&A] [answered]
In this guide: https://github.com/nodkz/mongodb-memory-server/blob/master/docs/guides/integration-examples.md
there is no reference of config.ts file, it is used in both globalSetup and globalTeardown files.
why there is no description of this file in this guide? should I just remove its implementation from the setup?

**Comment by hasezoey** [comment, marked as answer]
currently it just means a config file that exports the mentioned variables (it does not matter from where these variables come from), created #520
